api/src/main/java/org/apache/cloudstack/api/command/admin/vm/ListImportVMTasksCmd.java

```diff
@@ -75,11 +76,8 @@ public class ListImportVMTasksCmd extends BaseListCmd {
             description = "Conversion host of the importing task")
     private Long convertHostId;
 
-    @Parameter(name = ApiConstants.LIST_ALL, type = CommandType.BOOLEAN, description = "Whether to list all import tasks.")
-    private boolean listAll = false;
-
-    @Parameter(name = ApiConstants.SHOW_COMPLETED, type = CommandType.BOOLEAN, description = "Whether to list completed tasks.")
-    private boolean showCompleted = false;
+    @Parameter(name = ApiConstants.STATE, type = CommandType.STRING, description = "List tasks by status, valid options are: Running, Completed, Failed")
```

Comment:
should we add a since field?

Reply:
Thanks @Pearl1594, the since parameter is added on the API class entirely and it will use that one as all the parameters have been added on the same version (4.22)